### PR TITLE
Upgrade `flappi` to Ruby 2.7.7

### PR DESCRIPTION
**Pull request by alto**
In preparation for Ruby 3 eventually.

See [Asana ticket](https://app.asana.com/0/1202486819364117/1203917135924343).

### CODE REVIEW

File: node_modules/.yarn-integrity [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "systemParams": "darwin-arm64-102",
+  "modulesFolders": [],
+  "flags": [],
+  "linkedModules": [],
+  "topLevelPatterns": [],
+  "lockfileEntries": {},
+  "files": [],
+  "artifacts": {}
+}
```
Comment (fsanggang) [inline node_modules/.yarn-integrity:10]: Where did this come from?

Comment (alto) [reply node_modules/.yarn-integrity:10]: Hmm, indeed. My quick-finger `git add -A` missed that one. I will remove it.

**Review by fsanggang — APPROVED** [review]
💯

**Review by OBaddeley — APPROVED** [review]
🔢 👍🏼

**Pull request merged**